pages/cloudflare/migrate-from-0.3.mdx

```diff
@@ -0,0 +1,38 @@
+import { SITE } from '../../config';
```

Comment:
what about `migrate-from-0.3-to-0.4.mdx`?

Reply:
I'm not sure, I feel like the migration can omit the `to-` part, as that can confuse people? (forcing them to check what's the latest version etc...)?

So I'm not sure, it would be nice for consistency...

Please pick whatever you prefer and I'm happy to go with that 🙂

Reply:
You can rename the file but set a different title in the sidebar, but up to you

Reply:
done 🙂